### PR TITLE
Remove sleeps.

### PR DESCRIPTION
**Pull request by HeMan**
- the sleeps is not needed.

### CODE REVIEW

**Comment by OttoWinter** [comment]
I mean the sleeps have no use code-wise, they were solely intended to make the wizard feel a bit more "interactive" and a bit more like an actual wizard or actual person. The aim was to make the onboarding experience as friendly as possible. Of course the drawback of these sleeps is that going through the wizard is a bit slower. However, I think that nobody's going to go through the wizard more than say 5 times anyway. 

Maybe the sleep periods are a bit too long, but in general it's nicer to have the text appear in succession and not have a wizard that just starts with a huge text blob that you have to parse through first. Don't get me wrong, I'm also kind of on the edge of having these sleeps in the wizard or not, but ATM I think they're actually good. I'm open to suggestions/reasons why they should not be there though.

**Comment by HeMan** [comment]
I added a environment variable called QUICKWIZARD that disables the sleeps.

File: esphomeyaml/wizard.py [hunk 1]
```diff
@@ -71,6 +70,13 @@
 """
 
 
+if os.environ['QUICKWIZARD']:
```
Comment (OttoWinter) [inline esphomeyaml/wizard.py]: This results in a `KeyError` if `QUICKWIZARD` is not specified in the environment:

screen shot 2018-04-11 at 21 40 49

This should work:

```yaml
if os.getenv('QUICKWIZARD', False):
    def sleep(time):
        pass
else:
    from time import sleep
```

Also, I mean I don't really care, but the other environment variable I've defined for OTAs, `ESPHOMEYAML_OTA_HOST_PORT`, has the prefix `ESPHOMEYAML_` so I'd prefer if this environment variable would also start with that prefix for consistency.

**Review by OttoWinter — COMMENTED** [review]
Using environment variables is an excellent solution 👍

File: esphomeyaml/wizard.py (hunk 1 above)
Comment (HeMan) [reply esphomeyaml/wizard.py]: Lol! It shows how I tested it!

**Review by OttoWinter — APPROVED** [review]
Looks good! Thanks 🐳👍

**Pull request merged**